state_version.go

```diff
@@ -139,13 +139,14 @@ type StateVersionCreateOptions struct {
 	// Optional: Specifies the run to associate the state with.
 	Run *Run `jsonapi:"relation,run,omitempty"`
 
-	// Optional: The external, json representation of state data, base64 encoded.
-	// https://www.terraform.io/internals/json-format#state-representation
-	// Supplying this state representation can provide more details to the platform
+	// Optional: The external, json representation of state outputs, base64 encoded.
+	// https://www.terraform.io/internals/json-format#values-representation (the contents of the
+	// "outputs" attribute)
+	// Supplying detailed outputs will provide more details to the platform
```

Comment:
What do you think about rewording this? 
```suggestion
	// Optional: The external, json representation of state outputs, base64 encoded. Supplying this field 
	// will provide more detailed output type information to TFE.
	// For more information on the contents of this field: https://www.terraform.io/internals/json-format#values-representation
```